index.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+use PhpParser\Node\Stmt\Echo_;
+
+require_once 'src/ProductModel.php';
+require_once 'src/ProductViewHelper.php';
+
+$db = new PDO('mysql:host=db; dbname=dvdcollection', 'root', 'password');
+
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$productModel = new ProductModel($db);
+
+//var_dump($_POST) ;
+
+if (
+    isset($_POST['title']) &&
+    isset($_POST['description']) &&
+    isset($_POST['run_time']) &&
+    isset($_POST['image']) &&
+    isset($_POST['genre']) &&
+    isset($_POST['starring'])
+  ) 
+  {
+    $title = $_POST['title'];
+    $description = $_POST['description'];
+    $run_time = $_POST['run_time'];
+    $image = $_POST['image'];
+    $genre = $_POST['genre'];
+    $starring = $_POST['starring'];
+  
+
+        if (strlen($title) == 0 || strlen($title) <1) {
+            echo "Please enter a valid dvd title of at least 2 characters";
+        } if (strlen($description) == 0 || strlen($description) < 20) {
+            echo "Please enter a valid dvd description of at least 20 characters";
+        } if (is_int($run_time) == false) {
+            echo "Please enter a valid number";
+        } if (intval($genre) == 0 || intval($genre) > 5) {
+            echo "Please choose a genre from the drop down menu";
+        } if (intval($starring) == 0 || intval($starring) > 9) {
+            echo "Please either choose an actor from the drop down menu or add a new actor";
+        } if (strlen($image) == 0 || strlen($image) <10) {
+            echo "Please add a valid URL address";  
```

Comment:
It would be really cool if you could figure out some way of showing these errors under the form, or somewhere inside of the body tag.

Technically speaking, outputting messages outside ofthe html body tag is a bit naughty